README.md

```diff
@@ -29,6 +29,7 @@ import App from './components/App';
 ReactDOM.render(
   <SipProvider
     host="sip.example.com"
+    path="ws" // Path in socket URI (e.g. wss://sip.example.com:7443/ws); null bu default
```

Comment:
I guess it's just placing `path` after `port` now and should be good to go (unless @denisnikulin finds something else) 😅

Reply:
@kachkaev noticed this and fixed in the last commit ;)

Reply:
...or not...